lib/services/github_issues.rb

```diff
@@ -118,11 +118,18 @@ def receive_webhook
         # update the status
         diff[:workflow_status] = new_status if !new_status.nil? && new_status != resource.workflow_status.name
       end
-    when "closed", "opened", "reopened"
-      new_status = data.status_mapping.nil? ? nil : data.status_mapping[issue.state]
-      diff[:workflow_status] = new_status if !new_status.nil? && new_status != resource.workflow_status.id
-
-      if action == "opened" && !new_status.nil? && new_status == resource.workflow_status.id
+    when "closed", "reopened"
+      diff = diff_status_change(resource, issue, diff)
```

Comment:
`diff` isn't being used after assigned.

Reply:
It does get used later down in the function, github just happens to have snipped out the relevant lines: https://github.com/aha-app/aha-services/pull/140/files#diff-b07d6f1e798e3e599bc32c63f13879d1R142